crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -2203,6 +2203,13 @@ impl<'src> Parser<'src> {
                 )
             }
         } else {
+            // test_err ipython_escape_command_in_with_statement
+            // # parse_options: {"mode": "ipython"}
+            // with a,?b
+            //     pass
+            // with a,?b
+            //     x = 1
+            //     y = 2
```

Comment:
We should add some test cases that contains parentheses like the ones that I've mentioned in my comment